lua/DCS-gRPC/methods/controllers.lua

```diff
@@ -30,3 +30,63 @@ GRPC.methods.setAlarmState = function(params)
 
   return GRPC.success({})
 end
+
+GRPC.methods.getDetectedTargets = function(params)
+  local unit = Unit.getByName(params.unitName)
+  if unit == nil then
+    return GRPC.errorNotFound("Could not find unit with name '" .. params.unitName .. "'")
+  end
+
+  local controller = Unit.getController(unit)
+  local targets = controller:getDetectedTargets()
```

Comment:
We should add the ability to specify by which detection method targets should be looked for as documented in https://wiki.hoggitworld.com/view/DCS_func_getDetectedTargets . This will allow us to break up the monolithic "SHOW ME EVERYTHING IN THIS SIMULATION FRAME" call into a series of smaller ones that return smaller target lists that could be executed over multiple frames.. This could help with performance.

We should also aim to include all built-in method parameters in our APIs.

Reply:
Implemented